### DISCUSSION TITLE
Better simulator support for low resolutions

### DISCUSSION THREAD

**Discussion by LeahBreedenNyaruka** [nyaruka/rapidpro#4385] [Ideas]
A user made the following suggestions on improving the simulator: 

1) The bottom of the simulator is often cut off in the browser, requiring the window to adjusted and made smaller. Suggested moving the simulator a bit higher. 

2) The font color of the simulator is a very light gray, which is really difficult to read. Could we make it darker or allow users to choose the color?

3) Is it possible to make the simulator a floating frame so it can move around the screen to make different parts of the screen visible?

<img width="482" alt="Screen Shot 2023-03-06 at 1 58 55 PM" src="https://user-images.githubusercontent.com/60149838/223205112-746e554e-793e-4458-aef8-f653aa95ab0f.png">

**Comment by ericnewcomer** [comment]
Leaving this here for the other suggestions, but today we moved the simulator closer to the top on the new ui and bumped up the darkness of the info text.  I think moving would be a good idea, but that's a bit more work and I'm not sure how much we want to put into react these days.
